western/index.json

```diff
@@ -2085,5 +2085,501 @@
     "412:411 M+ 20:34:00 +19:30:00 20:34:00 +20:30:00 VUL DEL",
     "410:690 M+ 20:15:00 +20:30:00 20:15:00 +21:15:00 VUL SGE",
     "687:686 M+ 19:15:00 +19:10:00 19:15:00 +21:05:00 VUL SGE"
+  ],
+  "asterisms": [
+    {
```

Comment:
I would really like to see "source" or "reference" entries here and in many other SC parts (DSO names, star names), so that users may (later) be able to deselect unwanted sources.

Also, IMHO "Ray Helper" has always been a preliminary term. Is there some more "official" terminology to find? Do these RHs also have common names?

Reply:
We could easily add the source entry indeed, but it can be done in a later step I guess

Reply:
Source info is available in Modern SC. Don't drop now what is hard to retrieve later! (Of course, there is always a history in GIT.) It is certainly more relevant in the desktop version.

Reply:
For the international names of stars, dso etc.. the sources are all specified in the common_names.tab file.
Only the non-western constellations and also the asterisms are not in this file. I guess it's less criticalto list the sources for these since they are already listed in the sky culture description